src/NuGet.Clients/PackageManagement.VisualStudio/SolutionRestore/SolutionRestoreWorker.cs

```diff
@@ -53,7 +59,28 @@ internal sealed class SolutionRestoreWorker : ISolutionRestoreWorker, IDisposabl
             _serviceProvider = serviceProvider;
 
             _componentModel = _serviceProvider.GetService<SComponentModel, IComponentModel>();
+#if VS15
+            _solutionManager = _componentModel.GetService<IVsSolutionManager>();
```

Comment:
all mef imports/dependencies should be declared in one place as constructor parameters: for consistency, maintenance and testing.